ci/parse_ci_envs.sh

```diff
@@ -16,7 +16,7 @@ if [ -n "${STAGE_NAME:?}" ]; then
       : "${TARGET:=centos8}"
       ;;
     *Leap*15*|*leap15*|*opensuse15*|*sles15*)
-      : "${CHROOT_NAME:=opensuse-leap-15.1-x86_64}"
+      : "${CHROOT_NAME:=opensuse-leap-15.2-x86_64}"
       : "${TARGET:=leap15}"
```

Comment:
question on future support:   Can we generate/test  SLES15.1, 15.2 etc... rpms ?   Or will we only support single versions?

Reply:
It would be my assumption that 15.2 EOLs 15.1 just like EL 7.8 EOLs 7.7.  IOW, EL 7.7 is no longer supported the moment that 7.8 comes out (without a [very expensive] RHEL Z-stream contract at least) so I don't see why we should be supporting EOL'd distros.  At best, it increases our support foot-print.